### DISCUSSION TITLE
Not albe to integrate xtoken module

### DISCUSSION THREAD

**Discussion by herou** [open-web3-stack/open-runtime-module-library#670] [Q&A]
Hi, 

I am trying to integrante xtoken module but having this issue when tryin to compile: 
<img width="1210" alt="Screenshot 2021-12-20 at 21 12 30" src="https://user-images.githubusercontent.com/13895430/146826922-87542007-7ee8-4f40-869e-0d18c8b0e571.png">

Branch with changes: https://github.com/Kabocha-Network/cumulus/blob/v0.9.13-elio-xtokens/parachain-template/runtime/Cargo.toml

Thank you

**Comment by xlc** [comment]
You Cargo.toml is configured incorrectly. Mostly likely some of the deps have missing  `default-features = false` and `xxx/std`.

**Reply by herou** [reply]
@xlc FYI, my Cargo.toml may is not configured perfectly but I was able to execute cargo build successfully before implement xtoken module. I think the issue is the xtoken implementation. I added `"orml-xtokens/std",` in the array of std and now I am having a different issue from the one above. The new issue: 
<img width="1018" alt="Screenshot 2021-12-20 at 23 30 03" src="https://user-images.githubusercontent.com/13895430/146841243-c164a207-0cce-4c99-8370-e3870571e473.png">

To sum up, my changes in Cargo.toml are: 

```
[dependencies.orml-xtokens]
default-features = false
git = 'https://github.com/open-web3-stack/open-runtime-module-library'
branch = 'polkadot-v0.9.13'
version = '0.4.1-dev'
```

and 

```
"orml-xtokens/std",
```

Do you have any idea why having this issue? I have a mac m1 machine.
**Note**: As I explained above, the issue generated is because of xtoken module implementation. I had no previous issues.

**Reply by zjb0807** [reply]
Maybe you need to update dependencies.
https://github.com/paritytech/cumulus/commit/05cc5f0e2acacc18796f45ffa3c7b4626fd1046d

**Comment by xlc** [comment]
Unless you can point it out where the bug it is within xtokens otherwise I don't know what can be improved on xtokens side.

Your issue may be triggered by adding a new pallet, but it is an issue in your code. The compiler error message have indicated exactly what's the error.
